docs/src/tricks/parallelization.md

```diff
@@ -64,7 +64,7 @@ function estimate_time_per_scf_step(basis::PlaneWaveBasis)
     (time_per_FFT_per_grid_point
      * prod(basis.fft_size)
      * length(basis.kpoints)
-     * div(basis.model.n_electrons, DFTK.filled_occupation(basis.model), RoundUp)
+     * div(basis.model.ensemble.n_electrons, DFTK.filled_occupation(basis.model), RoundUp)
```

Comment:
I wonder if we should have a `n_electrons(model)` and an `n_electrons(basis)` to avoid such deep hierachies in this case.